forest_benchmarking/qubit_spectroscopy.py

```diff
@@ -739,7 +765,7 @@ def acquire_data_cz_phase_ramsey(qc: QuantumComputer,
     return pd.DataFrame(results)
 
 
-def estimate_cz_phase_ramsey(df: pd.DataFrame):
+def estimate_cz_phase_ramsey(df: pd.DataFrame):# -> List[Dict]:
```

Comment:
```suggestion
def estimate_cz_phase_ramsey(df: pd.DataFrame): -> List[Dict]:
```

Reply:
Ahh yes. I made everything else a DataFrame so I'll do that here too.